code/drasil-example/Drasil/Projectile/ProblemDescription.txt

```diff
@@ -0,0 +1,44 @@
+﻿Physical System Description
+
+Elements:
+	1) The object
+	2) The ground
+
+Assumptions
+1) The projectile motion is in 2D.
+2) The object is a point mass.
+3) The object's path starts and ends at ground level.
+4) The effects of drag, wind, and the Earth's curvature are ignored.
+5) The flight is free; there are no collisions during the trajectory of the object.
+6) The positive x-direction is the direction from start to the end of the object's path.
+7) The positive y-direction is the direction away from the ground.
+
+Goal Statments
+1) Check if the final position from user input is reached by the object with initial conditions.
+2) If the final position is not reached, provide the offset and if the object went long or short.
+
+Theoretical Models
+1) v = ds/dt
+2) a = dv/dt
+
+General Definitions
+1) v_f = v_i + a*t
+2) d = v_i_x*t + a_x*(t^2)/2
```

Comment:
You will need to add an assumption for constant acceleration, which implies an assumption that the sum of the unbalanced forces and the mass are constant.  If we had air-drag, which would add a force opposing the direction of flight, this general definition would no longer apply.

Reply:
@smiths Combining your proposed fix for splitting assumptions, what about:

"4) Acceleration is constant in the x-direction.
5) The effects of the Earth's curvature are ignored."
?

Reply:
I would say that the acceleration is zero in the x-direction, because there are no forces acting in this direction.  The acceleration is constant in the y-direction, as dictated by the value of acceleration due to gravity.

Reply:
That's what I thought but not what I typed 😂 final answer:

"4) Acceleration is zero in the x-direction.
5) Acceleration in the y-direction is only caused by gravity.
6) The effects of the Earth's curvature are ignored."

Reply:
That's a great start @samm82.  I'm sure we'll refine as we go along, but I like how the assumptions are shaping up.